src/app/assets/languages/en.json

```diff
@@ -443,7 +443,8 @@
       "no-templates-body": "Add cache templates to the server configuration.",
       "table-label": "Create caches from custom templates.",
       "no-encoding-warning": "This cache either has no encoding configuration or uses an encoding that the console does not support. You must use Hot Rod clients to perform read and write operations on this cache.",
-      "no-encoding-docs": "See \"Cache Encoding and Marshalling\" in the documentation."
+      "encoding-docs-message": "See \"Cache Encoding and Marshalling\" in the documentation.",
+      "pojo-encoding": "This cache uses \"{{encoding}}\" encoding. Data manipulation or displaying entries might not be fully supported from the Console."
```

Comment:
"{{brandname}} Console might not fully support data manipulation or displaying entries."
OR if I understand correctly 
"{{brandname}} Console might not fully support data manipulation or displaying entries for caches that don't use ProtoStream encoding."

Reply:
done